### PR TITLE
[bitnami/spring-cloud-dataflow] Revisit tests

### PR DESCRIPTION
**Pull request by FraPazGal**
Signed-off-by: FraPazGal <fdepaz@vmware.com>

### Description of the change

After reviewing the existing tests and the general testing strategy we have been following until now, we saw there was a need to revisit them. The main objective that our chart testing strategy should aim for is asserting that the Chart deliverable works. As such, the focus should not be on the application itself or the container, which are regarded as trustful components (that is, they should have been tested at a previous stage).

As charts usually pack a number of different components, it is essential to test that their integrations and the application itself work as expected.

Consequently, we have updated the tests so they are related to the chart deliverable and left only a couple of basic tests to verify that the application functions correctly. Besides that, an effort has been made to reduce future tests' flakiness improving some existing selectors and checks.

Tested in https://github.com/FraPazGal/charts/pull/107

### Benefits

Improved, revisited test strategy.

### Possible drawbacks

None

### Applicable issues

NA

### Additional information

NA

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/spring-cloud-dataflow/cypress/cypress/fixtures/imports.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "taskStarters": "Task application starters for Docker",
+  "streamStarters": "Stream application starters for RabbitMQ/Docker"
```
Comment (agomezmoron) [inline .vib/spring-cloud-dataflow/cypress/cypress/fixtures/imports.json:3]: I don't see where we use here those imports.json. Can we remove RabbitMQ string for another one if it's something to have a string? I don't like mixing product names.

**Review by agomezmoron — CHANGES_REQUESTED** [review]
I requested a minor change. In general, LGTM

File: .vib/spring-cloud-dataflow/cypress/cypress/fixtures/imports.json (hunk 1 above)
Comment (FraPazGal) [reply .vib/spring-cloud-dataflow/cypress/cypress/fixtures/imports.json:3]: This use case is a bit special. Spring CD includes some application starters not imported by default. These imported apps are needed to create tasks and streams, so we must do the import before starting our tests. These 2 strings are just used to select the correct option in the imports section:

Screenshot 2022-09-20 at 18 25 27

I could avoid having these strings in a fixture file (maybe as a `const` variable) but I left them there as that's how they were before the revisit.

Comment (agomezmoron) [reply .vib/spring-cloud-dataflow/cypress/cypress/fixtures/imports.json:3]: No worries @FraPazGal , I wasn't aware of that detail, so LGTM

**Review by agomezmoron — APPROVED** [review]

**Pull request merged**